File: Employee.java
import java.util.ArrayList;

public class Employee {
    private String name;
    private String lastname;
    private ArrayList<BankAccount> listAccounts = new ArrayList<>();

    public Employee() {
    }

    public Employee(String name, String lastname) {
        this.name = name;
        this.lastname = lastname;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getLastname() {
        return lastname;
    }

    public void setLastname(String lastname) {
        this.lastname = lastname;
    }

    public ArrayList<BankAccount> getListAccounts() {
        return listAccounts;
    }

    public void setListAccounts(ArrayList<BankAccount> listAccounts) {
        this.listAccounts = listAccounts;
    }
    //un statico puede estar entre otros paquetes sin afectar su seguridad
    public void addAccount(BankAccount newAccount){
        listAccounts.add(newAccount);
    }
}

File: Employee.py
class Employee:
    def __init__(self, name="", lastname=""):
        self.name = name
        self.lastname = lastname
        self.list_accounts = []

    def get_name(self):
        return self.name

    def set_name(self, name):
        self.name = name

    def get_lastname(self):
        return self.lastname

    def set_lastname(self, lastname):
        self.lastname = lastname

    def get_list_accounts(self):
        return self.list_accounts

    def set_list_accounts(self, list_accounts):
        self.list_accounts = list_accounts

    def add_account(self, new_account):
        self.list_accounts.append(new_account)
